Source/Core/UICommon/ResourcePack/ResourcePack.cpp

```diff
@@ -156,47 +188,40 @@ bool ResourcePack::Install(const std::string& path)
         break;
       }
     }
-
     if (provided_by_other_pack)
       continue;
 
-    if (unzLocateFile(file, ("textures/" + texture).c_str(), 0) != UNZ_OK)
-    {
-      m_error = "Failed to locate texture " + texture;
-      return false;
-    }
-
     const std::string texture_path = path + TEXTURE_PATH + texture;
-    std::string m_full_dir;
-    SplitPath(texture_path, &m_full_dir, nullptr, nullptr);
+    std::string texture_full_dir;
+    if (!SplitPath(texture_path, &texture_full_dir, nullptr, nullptr))
+      continue;
 
-    if (!File::CreateFullPath(m_full_dir))
+    if (!File::CreateFullPath(texture_full_dir))
     {
-      m_error = "Failed to create full path " + m_full_dir;
+      m_error = "Failed to create full path " + texture_full_dir;
```

Comment:
`m_error = fmt::format("Failed to create full path {}", texture_full_dir);`

Could do the others as well but some already exist :/

Reply:
tbh i dont see advantage of fmt for this case of joining 2 strings with no real formatting

Reply:
Yeah, it was never clear to me if there would be a win from avoiding the concatenation cost somehow with fmt.  But given this is an error case anyway, fine to leave it as is.